backend/isoko-backend/__tests__/unit/handlers/business/post-list-business.test.js

```diff
@@ -0,0 +1,282 @@
+const {
+    postListBusinessHandler,
+ } = require('../../../../src/handlers/business/post-list-business.js');
+ const dynamodb = require('aws-sdk/clients/dynamodb');
+ const { BUSINESS_TABLE } = require('../../../../src/constants');
+ 
+ jest.mock('aws-sdk/clients/dynamodb');
+ 
+ describe('PostListBusinessHandler tests', () => {
+    let putSpy;
+ 
+    beforeAll(() => {
+       putSpy = jest.spyOn(dynamodb.DocumentClient.prototype, 'put');
+    });
+ 
+    afterAll(() => {
+       putSpy.mockRestore();
+    });
+
+    describe('Invalid query param tests', () => {
+        it('Should throw an error when wrong HTTP method is used', async () => {
+            // arrange
+            const event = {
+            httpMethod: 'GET',
+            };
+
+            // act
+
+            // assert
+            await expect(async () => {
+            await postListBusinessHandler(event);
+            }).rejects.toThrowError();
+        });
+    }); 
+
+    describe('Valid input tests', () => {
+        it('Should post business info with all optional params included and return details', async () => {
+            // arrange
+            mockPutResults = 
+            {
+                // should this include pk/sk? 
+                //pk: 'BUSID1',
```

Comment:
Was working on the post review task and I had the same question. Shouldn't pk and sk be a part of the item because that is what we are adding to our dynamodb table.

Reply:
you 100% right. fixed in latest commit!